controllers/mod/artefacts_data.rb

```diff
@@ -0,0 +1,71 @@
+class ArtefactsdataController < ApplicationController
+  namespace "/artefacts/:artefactID/resources" do
+    
+    get do
+      ontology, latest_submission = get_ontology_and_latest_submission
+      check_access(ontology)
+      _, page, size = settings_params(LinkedData::Models::Class).first(3)
+      size_per_type = [size / 6, 1].max
```

Comment:
why this?

Reply:
this is the logic to get from each type, the same number of resources. if the pagesize < 6, we will get only 1 resources from each type

<img src="https://github.com/user-attachments/assets/e2be39e9-9745-442e-8ec8-1110a13d05a2" width="200">

Reply:
We need to see when deployed how is this working.